internal/pkg/manifest/env.go

```diff
@@ -415,6 +416,70 @@ func (cfg *EnvironmentHTTPConfig) loadLBConfig(env *config.CustomizeEnv) {
 type PublicHTTPConfig struct {
 	SecurityGroupConfig ALBSecurityGroupsConfig `yaml:"security_groups,omitempty"`
 	Certificates        []string                `yaml:"certificates,omitempty"`
+	ELBAccessLogs       ELBAccessLogsArgsOrBool `yaml:"access_logs,omitempty"`
+}
+
+// ELBAccessLogsArgsOrBool is a custom type which supports unmarshaling yaml which
+// can either be of type bool or type ELBAccessLogsArgs.
+type ELBAccessLogsArgsOrBool struct {
+	EnableAccessLogs  *bool
+	ELBAccessLogsArgs ELBAccessLogsArgs
```

Comment:
nit:
```suggestion
	Enabled  *bool
	AdvancedConfig ELBAccessLogsArgs
```
Hopefully it reads better this way! I think the `"AccessLogs"` in `EnableAccessLogs` and `ELBAccessLogsArgs` are sort of repetitive information because those keywords are already in the struct name.